fapolicy_analyzer/glade/policy_rules_admin_page.glade

```diff
@@ -2,14 +2,16 @@
 <!-- Generated with glade 3.38.2 -->
 <interface>
   <requires lib="gtk+" version="3.22"/>
-  <object class="GtkBox" id="policyRulesAdminPage">
+  <!-- n-columns=3 n-rows=3 -->
```

Comment:
In glade you have extra rows that don't seem to be used. Update the properties to remove those.
![image](https://user-images.githubusercontent.com/17859684/143040793-067cb521-ddc6-4646-9635-2c8f25c2dba0.png)

Reply:
Fixed by transferring back to a Box object